src/test/testing_assert.cpp

```diff
@@ -13,27 +13,41 @@
 #include "storage/table.hpp"
 #include "storage/value_column.hpp"
 
+#define ANSI_COLOR_RED "\x1B[31m"
+#define ANSI_COLOR_GREEN "\x1B[32m"
+#define ANSI_COLOR_BG_RED "\x1B[41m"
+#define ANSI_COLOR_BG_GREEN "\x1B[42m"
+#define ANSI_COLOR_RESET "\x1B[0m"
+
+#define EPSILON 0.0001
+
 namespace {
 
 using Matrix = std::vector<std::vector<opossum::AllTypeVariant>>;
 
 Matrix _table_to_matrix(const std::shared_ptr<const opossum::Table>& t) {
-  // initialize matrix with table sizes
-  Matrix matrix(t->row_count(), std::vector<opossum::AllTypeVariant>(t->column_count()));
+  // initialize matrix with table sizes, including column names/types
+  Matrix matrix(t->row_count() + 2, std::vector<opossum::AllTypeVariant>(t->column_count()));
```

Comment:
Missed this in the first review: Please write out "table". While it is clear in this case, we don't want to let abbreviations like this sneak in.

Reply:
(same for col_id)